### DISCUSSION TITLE
how to publish package without terminal?

### DISCUSSION THREAD

**Discussion by LevaCompany** [community/community#38377] [Actions]
how to publish package without terminal?

**Comment by ossanna16** [comment]
Hi there @LevaCompany and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our official introduction thread
To connect with other new users, check out this Discussion to Meet Your GitHub Community Team, further your GitHub knowledge by working through our GitHub Skill 🆙 exercises, and participate in our weekly Fun Friday chats

**Comment by EliasAfara** [comment]
Hey @LevaCompany check out [here](https://www.youtube.com/watch?v=JZVGKU7YYAA&ab_channel=GitHub) how to publishing to gitHub packages with actions is done.
